### PR TITLE
MP/ Adapt username request to styleguide

### PR DESCRIPTION
**Pull request by MPucheta**

### CODE REVIEW

File: src/screens/authentication/UsernameRequest/index.js [hunk 1]
```diff
@@ -1,22 +1,61 @@
...
+  } else {
+    return (
+      <PrimaryButton
+        onPress={onPress}
+      >
+        <Description white>Continuar</Description>
+      </PrimaryButton>
+    )
+  }
+}
```
Comment (nahudalla) [inline src/screens/authentication/UsernameRequest/index.js]: The `value` prop is only used for the `if` statement?

File: src/screens/authentication/UsernameRequest/index.js [hunk 2]
```diff
@@ -62,31 +97,31 @@ const UsernameRequest = (props) => {
...
-          source={DrCannabis}
-        />
-        <Text fontVariant='description' style={styles.information}>Antes de iniciar sesión vamos a necesitar un nombre para tu usuario.</Text>
-        <Text fontVariant='h1' style={styles.usernameTag}>Usuario</Text>
+        <Subtitle style={styles.title}>Creá tu nombre de usuario</Subtitle>
+        <Description
+          gray
+          style={styles.description}
+        >
+        Escribí un nombre de usuario para identificarte dentro de la plataforma
```
Comment (nahudalla) [inline src/screens/authentication/UsernameRequest/index.js]: Wrong indentation here.

**Review by nahudalla — CHANGES_REQUESTED** [review]

File: src/screens/authentication/UsernameRequest/index.js (hunk 1 above)
Comment (MPucheta) [reply src/screens/authentication/UsernameRequest/index.js]: Yes

File: src/screens/authentication/UsernameRequest/index.js (hunk 2 above)
Comment (MPucheta) [reply src/screens/authentication/UsernameRequest/index.js]: Standard did not mark it. Should I put the text two spaces to the right?

Comment (nahudalla) [reply src/screens/authentication/UsernameRequest/index.js]: Yes, I like increasing indentation inside every tag. I don't know if standard has a rule for that.

File: src/screens/authentication/UsernameRequest/index.js (hunk 1 above)
Comment (nahudalla) [reply src/screens/authentication/UsernameRequest/index.js]: :thinking: I think that prop should be a `boolean` with a more descriptive name, like `disabled`.

**Review by nahudalla — APPROVED** [review]

**Pull request merged**